channels/de.m3u

```diff
@@ -316,11 +316,11 @@ https://cldf-hlsgw.r53.cdn.tv1.eu/1000153copo/hk1.m3u8
 https://c13014-l-hls.u.core.cdn.streamfarm.net/1000153copo/hk2.m3u8
 #EXTINF:-1 tvg-id="Parlamentsfernsehen2.de" tvg-name="Parlamentsfernsehen 2" tvg-country="DE" tvg-language="German" tvg-logo="" group-title="",Parlamentsfernsehen 2 (1080p)
 https://cldf-hlsgw.r53.cdn.tv1.eu/1000153copo/hk2.m3u8
-#EXTINF:-1 tvg-id="PearlTV.de" tvg-name="Pearl TV" tvg-country="DE" tvg-language="" tvg-logo="" group-title="",Pearl TV [Offline]
+#EXTINF:-1 tvg-id="PearlTV.de" tvg-name="Pearl TV" tvg-country="DE" tvg-language="German" tvg-logo="https://raw.githubusercontent.com/jnk22/kodinerds-iptv/master/logos/tv/pearltv.png" group-title="",Pearl TV [Offline]
 http://enstyle-live.hls.adaptive.level3.net/ses/enstyle/stream1/streamPlaylist.m3u8
```

Comment:
https://linkastream.co/headless?url=https://www.youtube.com/PEARLTV4K/live

Reply:
![image](https://user-images.githubusercontent.com/7253922/129435875-fc1b8d5f-f46f-4b6b-b28b-49bfa29912f7.png)

Reply:
yes, the linkastream service isn't back online yet.